src/streams/logs.ts

```diff
@@ -0,0 +1,69 @@
+import { Readable } from 'stream';
+import { listOnePageLogResources } from '../api/logs';
+import { LogApiResource, Sid, GotClient } from '../types';
+import { LogsConfig } from '../types/logs';
+
+export class LogsStream extends Readable {
```

Comment:
This generally looks good but what bothers me is that I cannot find a way to type what "data" we are sending on the `data` event. I'll ask Twitter about this.

Reply:
This is a good point. Will share the tweet too.

Reply:
Hmm. It looks like the `Readable` definition types the emitted object from the event as `Buffer | string` and I'm sending a (JSON encoded) string in this case. I'm really not sure how you'd make that more specific without reimplementing chunks of the underlying class, which I'd prefer not to do.

Reply:
I realised that I was defining the `Readable` to be in `objectMode` so I can `push` JavaScript objects through the stream, but I still can't see any way to type the object emitted by the event.

I'm not sure if we can do much more with it than that.